libs/librrgraph/CMakeLists.txt

```diff
@@ -13,16 +14,27 @@ add_library(librrgraph STATIC
     ${LIB_SOURCES}
 )
 
-target_include_directories(librrgraph PUBLIC ${LIB_INCLUDE_DIRS})
-
+target_include_directories(librrgraph PUBLIC  ${PROJECT_SOURCE_DIR}/../../vpr/ )
+target_include_directories(librrgraph PUBLIC  ${PROJECT_SOURCE_DIR}/../EXTERNAL/ )
```

Comment:
Please avoid to create a loop between VPR and this library. This library should not have any dependency on VPR. It should be self-contained.

In order to do this, you need to move all the related header and source files to the ``librrgraph``

Therefore, I think this line should not be modified.